services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dto.{ FileContent, ValidateError }
+import cromwell.pipeline.womtool.WomTool
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class ProjectFileService(womTool: WomTool)(implicit executionContext: ExecutionContext) {
```

Comment:
I've just noticed that `WomTool` is an implementation of `WomToolApi`. This is cool and it means that `ProjectFileService` must take `WomToolApi`, not `WomTool`. In that case, we can easily switch the implementation if we need to.

Reply:
Yes, my bad, thank you